src/utils/apis/auth.js

```diff
@@ -0,0 +1,48 @@
+export const BASE_URL =
+  process.env.NODE_ENV === "production"
+    ? "https://api.haikukedoodle.com"
+    : "http://localhost:3001";
+
+const handleAuthResponse = (res) => {
```

Comment:
Please, don't duplicate `handleResponse` in every api file. You need to create it only once with a neutral name `handleResponse` and import to any file where you need to check responses